src/AudioStreamer.js

```diff
@@ -124,7 +125,7 @@ var ScarletsAudioStreamer = function(chunksDuration){
 			scope.bufferElement[index].connect(scope.outputNode);
 
 		else // Direct output to destination
-			scope.bufferElement[index].connect(scope.audioContext.destination);
+			scope.bufferElement[index].connect(scope.audioContext.destinatgion);
```

Comment:
I think it should be `.destination` instead of `.destinatgion` 😄 
It's connecting the [AudioBuffer node](https://developer.mozilla.org/en-US/docs/Web/API/BaseAudioContext/createBufferSource) to [AudioContext](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext#examples)'s destination if the stream is not using Plugin/Effect

Reply:
What a shame! Sorry